map_gen/maps/crash_site/outpost_builder.lua

```diff
@@ -1213,6 +1219,29 @@ local function do_artillery_turrets_targets()
     end
 end
 
+local function set_pollution_multiplier(args, player)
+    local multiplier = tonumber(args.multiplier)
+    if not multiplier then
+        player.print("Fail")
+        return
+    end
+
+    local old_multiplier = pollution_multiplier.value
+    pollution_multiplier.value = multiplier
+    local message = player.name..' changed magic crafter pollution multiplier from '..old_multiplier..' to '..pollution_multiplier
+    for _, p in pairs(game.players) do
+        if p.admin then
+            p.print(message)
+        end
+    end
+end
+
+local server_player = {name = '<server>', print = print}
+local function get_pollution_multiplier(player)
```

Comment:
This doesn't work. I think it should be.
```suggestion
local function get_pollution_multiplier(args, player)
```

Reply:
Updated in next commit